script.js

```diff
@@ -19,13 +19,26 @@ restaurantsApp.getRestaurantIDs = (searchTerm, searchLocation) => {
             location: searchLocation
         }
     }).then(function (result) {
+        //Empty restaurantList so we can append results each time
+        $('.restaurantList').empty()
         //This takes the first three results, can be changed later
         for (let i = 0; i < 3; i++) {
             let businessID = result.businesses[i].id
             let businessName = result.businesses[i].name
+            let businessImage = result.businesses[i].image_url
             let businessRating = result.businesses[i].rating
             let businessPrice = result.businesses[i].price
             let businessAddress = result.businesses[i].location.display_address
+
+            let html = `<div>
+                <img src="${businessImage}">
+                <h2>${businessName}</h2>
+                <span>${businessRating}</span>
+                <span><i class="fas fa-dollar-sign"></i><i class="fas fa-dollar-sign"></i></span> 
+                <h3>${businessAddress}</h3>
+                </div>`
+            //Displays each result to the page
+            $('.restaurantList').append(html)
```

Comment:
usually in our code alongs we did separate function for displaying info on the page, however you have wrote the display code inside the .then method of the first ajax call. that is why I created an array for the info to be accessible outside the .then method of the first ajax call

Reply:
made your changes! take a look and let me know what you think